src/main/java/com/odysseusinc/arachne/portal/api/v1/controller/BaseAnalysisController.java

```diff
@@ -206,6 +207,28 @@ public BaseAnalysisController(BaseAnalysisService analysisService,
         return result;
     }
 
+    @ApiOperation("Get short analysis info.")
+    @RequestMapping(value = "/api/v1/analysis-management/analyses/{analysisId}/short", method = GET)
+    public JsonResult<ShortBaseAnalysisDTO> getShortAnalysis(
+            Principal principal,
+            @PathVariable("analysisId") Long analysisId)
+            throws PermissionDeniedException, NotExistException, NotUniqueException {
+
+        JsonResult<ShortBaseAnalysisDTO> result;
+        User user = getUser(principal);
+        if (analysisId == null) {
```

Comment:
As I remember, the pathVariable cannot be null. But even if it can - you should be able to replace this check with annotation for method parameter